_Sidebar.md

```diff
@@ -37,13 +37,13 @@
     * [[How to write type definitions|Guide-on-defining-types]]
     * [[How to write design docs|Writing-design-docs]]
     * [[Revert and Regression Policy|Revert-and-Regression-Policy]]
-    * [[If your build fails...|If-Your-Build-Fails]]
+    * Debugging
+      * [[Diagnose PR failures|Diagnose-PR-failures]]
```

Comment:
The title of this page seems a bit off because it's not the PR that's failing. Maybe something like "What to do if CI checks on PRs fail" or "What to do if there's a red cross next to your PR"?

Reply:
Changed to `If-CI-checks-fail-on-your-PR.md`